src/module.c

```diff
@@ -185,6 +185,17 @@ static int parseRangeArguments(RedisModuleCtx *ctx, Series *series, int start_in
     return REDISMODULE_OK;
 }
 
+static int parseCountArgument(RedisModuleCtx *ctx, RedisModuleString **argv, int argc, long long *count) {
```

Comment:
this should be part of parseRangeArguments

Reply:
It can't be since parseRangeArguments doesn't receive argc which is required for RMUtil_ArgIndex. Adding it will just abuse the existing function. 
Currently, range, filters and aggregations are parsed separately which is a good design!